packages/container/tests/test-helpers/build-owner.js

```diff
@@ -7,7 +7,7 @@ export default function buildOwner(props) {
   let Owner = EmberObject.extend(RegistryProxy, ContainerProxy, {
     init() {
       this._super(...arguments);
-      let registry = this.__registry__ = new Registry();
+      let registry = this.__registry__ = new Registry(this._registryOptions);
```

Comment:
This entire file is only for tests (it is a testing helper that is imported via `import buildOwner from 'container/tests/test-helpers/build-owner'`).